content/set-environment.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
Given how short and direct this is, can we include this directly in `wp-config`?